README.md

```diff
@@ -1,90 +1,83 @@
-* [Samples](https://github.com/microsoft/WindowsAppSDK-Samples) - Discover Windows App SDK samples to assist in learning how to use the latest features!
-
-### Contributing
-
-[![Bugs](https://img.shields.io/github/issues/microsoft/windowsappsdk/bug)](https://github.com/microsoft/WindowsAppSDK/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
-
-_We want to hear from you!_
-
-File a [new issue!](https://github.com/microsoft/WindowsAppSDK/issues/new/choose) Tell us what problem you're
-trying to solve, how you've tried to solve it so far, and what would be the ideal solution for your app. Bonus
-points if there's a gist or existing repo we can look at with you.
+<p align="center">
+  <a href="https://aka.ms/winappsdk">
+    <img src="./docs/images/header.png" alt="WinUI Banner" />
+  </a>
+</p>
+
+<h1 align="center">Windows App SDK</h1>
+
+<h3 align="center">
+  <a href="https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk">About Windows App SDK</a>
+  <span> · </span>
+  <a href="https://learn.microsoft.com/windows/apps/desktop">Documentation</a>
+  <span> · </span>
+  <a href="https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/release-channels">Release notes</a>
+  <span> · </span>
+  <a href="https://github.com/microsoft/WindowsAppSDK-Samples">Samples</a>
+</h3>
+
+<p align="center">
+  <a style="text-decoration:none" href="https://www.nuget.org/packages/Microsoft.WindowsAppSDK">
+    <img src="https://img.shields.io/nuget/v/Microsoft.WindowsAppSDK" alt="WindowsAppSDK NuGet" /></a>
+</p>
+
+[Windows App SDK](https://aka.ms/winappsdk) (F.K.A. Project Reunion) is a set of libraries, frameworks, components, and tools that you can use in your apps to access powerful Windows platform functionality from all kinds of apps on many versions of Windows. The Windows App SDK combines the powers of Win32 native applications alongside modern API usage techniques, so your apps light up everywhere your users are.
+
+- **WinUI 3 support:** [WinUI](https://aka.ms/winui) embodies Fluent Design to enable intuitive, accessible, and powerful experiences and the latest user interface patterns.
+- **Access modern features:** [Modern resource tooling](https://github.com/microsoft/WindowsAppSDK/issues/11), [modern lifecycle helpers](https://github.com/microsoft/WindowsAppSDK/issues/9), [startup tasks](https://github.com/microsoft/WindowsAppSDK/issues/10), and more without having to rewrite your app.
+- **Backwards compatibility:** Down to Windows 10 1809 (build 17763). There may be some API that are dependent on new OS features
+(like new Action Center functionality), however we will do our best to ensure this is the exception and not the norm, and provide reasonable fallbacks when possible.
+- **Wide platform support:** Win32, WPF, WinForms, and more.
+- **MSIX support:** You can still use your current installer, but there are [many benefits](https://docs.microsoft.com/windows/msix/overview#key-features) to using MSIX.
```

Comment:
Nit: I think the main point is that developers are not bound to MSIX and can use their own thing. I think the main high-light should be about that devs can use their own installers? But yes, we do want to call out MSIX

Reply:
Done